### DISCUSSION TITLE
LG Screen Manager vs. LG OnScreen Control?

### DISCUSSION THREAD

**Discussion by JBallin** [Homebrew/discussions#1874] [Casks]
The LG Screen Manager cask was removed in https://github.com/Homebrew/homebrew-cask-drivers/pull/523, given that LG's site recommends installing the app directly from the [app store](https://apps.apple.com/us/app/lg-screen-manager-lg-monitor/id1142051783?mt=12).

In contrast, LG OnScreen Control has a [cask](https://github.com/Homebrew/homebrew-cask-drivers/blob/master/Casks/lg-onscreen-control.rb), and is not on the app store.

Does anyone know the difference between the two apps? My assumption is that OnScreen Control has all the functionality present in Screen Manager, which I think we could clarify either as a comment in source or in the description of the cask - but I first wanted to confirm this assumption here.

Thanks!

**Comment by mjgardner** [comment]
LG Screen Manager is for updating monitor firmware.

LG OnScreen Control is for dividing the monitor screen into split areas that individual windows can snap into. On a PC, it also provides picture and orientation controls as well as firmware update, but the Mac version does not provide the latter. So your assumption that it has Screen Manager’s functionality is incorrect.
